release-notes/release-notes59.md

```diff
@@ -80,6 +80,16 @@ The {{site.productname}} 5.9 release includes an accompanying release of the **<
 
 For information on the <<Premium Plugin Name>> plugin, see: [<<Premium Plugin Name>> plugin]({{site.baseurl}}/plugins/<<Premium Plugin Name>>/).
 
+### PowerPaste 5.6.0
+
+The {{site.productname}} 5.9 release includes an accompanying release of the **PowerPaste** premium plugin.
+
+**PowerPaste** 5.6.0 introduces the following enhancements:
+
+- Asynchronous callback functionality added to the `powerpaste_word_import`, `powerpaste_googledocs_import` and `powerpaste_html_import` options. This allows the integrator to dynamically choose the paste mode on each paste event. It is designed to be used to replicate the `prompt` dialog with custom, non-TinyMCE UI.
```

Comment:
```suggestion
- Asynchronous callback functionality added to the `powerpaste_word_import`, `powerpaste_googledocs_import` and `powerpaste_html_import` options. This allows the paste mode to be dynamically chosen on each paste event. It is designed to be used to replicate the `prompt` dialog with custom, non-TinyMCE UI.
```
I don't think I've ever seen us say `the integrator` especially when they are the target audience.

Reply:
Me neither, but I couldn't think of anything better yesterday. Thanks!

Reply:
`now possible` -> `now valid values` or something? That part reads oddly to me...